<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyThreadSignalTest</class>
 <widget class="QWidget" name="MyThreadSignalTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyThreadSignalTest</string>
  </property>
  <widget class="QPushButton" name="thread1Btn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Thraed 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="thread2Btn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Thread 2</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="thread1CheckBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Thread 1 Slot Action</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="thread2CheckBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Thread 2 Slot Action</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendSignalBtnAuto">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Send Signal Auto</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendQueuedSignalBtn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Send Signal Queued</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
